ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -289,7 +294,11 @@ private void updateClippingToRect(Rect clippingRect) {
 
   private void updateSubviewClipStatus(Rect clippingRect, int idx, int clippedSoFar) {
     View child = Assertions.assertNotNull(mAllChildren)[idx];
-    sHelperRect.set(child.getLeft(), child.getTop(), child.getRight(), child.getBottom());
+    sHelperRect.set(
+      child.getLeft() + (int) child.getTranslationX(),
```

Comment:
Actually I changed it to (floor, floor, ceil, ceil) that way it keeps the same behaviour for non decimal numbers and will still make sure to never clip views that end up with decimal bounds after transform.